climpred/metrics.py

```diff
@@ -41,19 +43,102 @@ def _get_norm_factor(comparison):
         KeyError: if comparison is not matching.
 
     """
-    comparison_name = comparison.__name__
-    if comparison_name in ['_m2e', '_e2c', '_e2r']:
+    if comparison.name in ['m2e', 'e2c', 'e2r']:
         fac = 1
-    elif comparison_name in ['_m2c', '_m2m', '_m2r']:
+    elif comparison.name in ['m2c', 'm2m', 'm2r']:
```

Comment:
Maybe comment goes here as well explaining why we need this fac. Or not, if it's well described in docs.